File: Paradigmas/Atividades/Atividade IX - Lambda, Matrizes e Sudoku/lambda/ex2.hs
main = do
    putStrLn "Digite a nota a:"
    a <- readLn :: IO Double
    putStrLn "Digite a nota b:"
    b <- readLn :: IO Double
    putStrLn "Digite a nota c:"
    c <- readLn :: IO Double
    let media = \a b c -> (a + b + c) / 3
        aprovado = \m -> m >= 6
    putStrLn $ "O aluno está " ++ (if aprovado (media a b c) then "aprovado" else "reprovado")

File: Paradigmas/Atividades/Atividade III/Ex2.py
verificar_aprovacao = lambda a, b, c: "Aprovado" if (a + b + c) / 3 >= 6 else "Reprovado"

a, b, c = map(float, input("Digite as três notas: ").split())

resultado = verificar_aprovacao(a, b, c)
print("O aluno está", resultado)
